### DISCUSSION TITLE
Not displayed in the AirPlay menu

### DISCUSSION THREAD

**Discussion by noraworld** [mikebrady/shairport-sync#1668] [Q&A]
I installed `shairport-sync` on my Raspberry Pi, but my iPhone and Mac don't show it on the AirPlay menu, even though the process is starting correctly.

I have read through [the Troubleshooting page](https://github.com/mikebrady/shairport-sync/blob/f5c4b51da827a7f8d9a72a1b6f986807aba47bfc/TROUBLESHOOTING.md), but I have no clue why it doesn't work.

I'm not sure what information is needed to resolve this problem, so it would be appreciated if you could teach me that.

**Comment by mikebrady** [comment]
This is not really a discussion item -- it's a troubleshooting issue, I'd suggest. Anyway, please take a look at [this](https://github.com/mikebrady/shairport-sync/issues/1047#issuecomment-664943215) sequence, where some detail on checking why the service doesn't seem to "appear".

**Comment by noraworld** [comment]
The issue you mentioned made me notice that some of the needed packages were missing. My iPhone can now connect to AirPlay on my Raspberry Pi. Thank you for helping me out!

But I have another problem. The sound on my iPhone plays by itself even though it's connected to AirPlay. It doesn't mean no sound plays; the iPhone plays the sound as if it's not connected to AirPlay.

How can I resolve this?

**Reply by mikebrady** [reply]
I'm not sure I quite understand. Can I just verify that you're selecting [AirPlay audio output correctly](https://support.apple.com/en-us/HT202809)?

**Reply by noraworld** [reply]
@mikebrady I'm sorry for the ambiguous explanation. After a few adjustments, I'm now able to hear the sound to a certain extent. But I still can't listen to the sound in some apps, such as Siri's voice, voice-over sound, or something like that. Is it possible to hear those kinds of sounds via AirPlay?

And I have another question. Can I listen to the sound on my Mac and iPhone at the same time?

**Comment by mikebrady** [comment]
Thanks for the update. Yes, some sounds do not come out on AirPlay, such as spoken directions from Apple Maps and maybe so some other things. With AirPlay 2, you can make and receive calls on iOS without disturbing the flow of music, which is a nice feature. 

You can play to Shairport Sync from iOS or from Mac, but only one at a time. Is that what you mean?

**Reply by noraworld** [reply]
Thank you for your reply, and I'm so sorry for the very late reply. It's been a while since I started to use AirPlay, and I'm having some issues, such as being unable to connect to AirPlay very frequently and taking a lot of time to start playing music or a movie. But this is off-topic with this issue, so I'm going to close it because the problem I mentioned for the first time has been resolved.
